<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Chenlin Lu </title> <meta name="author" content="Chenlin Lu"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://lucl13.github.io/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Chenlin</span> Lu </h1> <p class="desc"></p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/chenlin_head-480.webp 480w,/assets/img/chenlin_head-800.webp 800w,/assets/img/chenlin_head-1400.webp 1400w," type="image/webp" sizes="(min-width: 930px) 270.0px, (min-width: 576px) 30vw, 95vw"> <img src="/assets/img/chenlin_head.jpg?2a156d96846b3d647f2e85837f158a6b" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="chenlin_head.jpg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> <div class="more-info"> <p>701 W. 168th St.</p> <p>New York, NY 10032</p> </div> </div> <div class="clearfix"> <p>I grew up in a small town called <a href="https://www.google.com/maps/place/Xihaizhen,+Haiyan+County,+Qinghai,+Haibei,+Qinghai,+China,+810299/@33.0747586,81.9877127,4z/data=!4m6!3m5!1s0x364d5cb1c9e60669:0x4d7557eefa77e26a!8m2!3d36.957868!4d100.902973!16s%2Fg%2F1pv0kg7w?entry=ttu&amp;g_ep=EgoyMDI1MDQxNi4xIKXMDSoJLDEwMjExNDU1SAFQAw%3D%3D" rel="external nofollow noopener" target="_blank">Xihai</a> in China, where I spent a joyful childhood immersed in nature. I completed both my undergraduate and graduate studies at <a href="https://www.tsinghua.edu.cn/en/" rel="external nofollow noopener" target="_blank">Tsinghua University</a>, where I became fascinated by the dynamic behavior of proteins—the remarkable molecular machines that sustain life. During my Ph.D., I spent a year in Tokyo as a visiting student, where I solved my first protein structures (<a href="https://www.rcsb.org/structure/7VIU" rel="external nofollow noopener" target="_blank">7VIU</a>) and enjoyed the city’s vibrant mix of science and culture.</p> <p>I am currently a postdoctoral fellow in the <a href="https://glasgowlab.org" rel="external nofollow noopener" target="_blank">Glasgow Lab</a> at Columbia University, continuing my journey to explore protein dynamics and conformational landscapes. My research lies at the intersection of computational and experimental biophysics, combined with machine learning, to understand allosteric regulation and to advance the de novo design of allosteric proteins.</p> <p>I’m developing methods to push the limits of HX/MS, enabling residue-level resolution of protein dynamics and uncovering allosteric regulation hidden from traditional structural techniques. These insights are guiding my long-term goal: designing allosteric proteins with controllable functions.</p> </div> <h2> <a href="/news/" style="color: inherit">news</a> </h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">Apr 19, 2025</th> <td> Chenlin’s academic page is now online! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20"> </td> </tr> </table> </div> </div> <h2> <a href="/publications/" style="color: inherit">selected publications</a> </h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/04_fr_4fold-Au.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="04_fr_4fold-Au.png" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lu_design_2022" class="col-sm-8"> <div class="title">Design of a gold clustering site in an engineered apo-ferritin cage</div> <div class="author"> <em>Chenlin Lu</em>, Basudev Maity, Xue Peng, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Nozomi Ito, Satoshi Abe, Xiang Sheng, Takafumi Ueno, Diannan Lu' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Communications Chemistry</em>, Dec 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s42004-022-00651-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract Water-soluble and biocompatible protein-protected gold nanoclusters (Au NCs) hold great promise for numerous applications. However, design and precise regulation of their structure at an atomic level remain challenging. Herein, we have engineered and constructed a gold clustering site at the 4-fold symmetric axis channel of the apo-ferritin cage. Using a series of X-ray crystal structures, we evaluated the stepwise accumulation process of Au ions into the cage and the formation of a multinuclear Au cluster in our designed cavity. We also disclosed the role of key residues in the metal accumulation process. X-ray crystal structures in combination with quantum chemical (QC) calculation revealed a unique Au clustering site with up to 12 Au atoms positions in the cavity. Moreover, the structure of the gold nanocluster was precisely tuned by the dosage of the Au precursor. As the gold concentration increases, the number of Au atoms position at the clustering site increases from 8 to 12, and a structural rearrangement was observed at a higher Au concentration. Furthermore, the binding affinity order of the four Au binding sites on apo-ferritin was unveiled with a stepwise increase of Au precursor concentration.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/07_FrAu_ArM.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="07_FrAu_ArM.png" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lu_novel_2023" class="col-sm-8"> <div class="title">Novel Au(I)-Based Artificial Metallo-Cycloisomerase for Catalyzing the Cycloisomerization of γ-Alkynoic Acids</div> <div class="author"> <em>Chenlin Lu</em>, Xue Peng, Basudev Maity, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Xiang Sheng, Yinhuan Zhou, Takafumi Ueno, Zheng Liu, Diannan Lu' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>ACS Catalysis</em>, Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/acscatal.3c01197" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Artificial metalloenzymes, which are designed rationally as hybrids of proteins and catalytically active transition-metal complexes, have become a promising approach for catalyzing unprecedented reactions for natural enzymes. In this study, we described the design and synthesis of an artificial metalloenzyme, a cycloisomerase that utilizes Au(I) incorporated into an apo-ferritin cage (Fr−Au) to efficiently catalyze the cycloisomerization of alkynoic acids, with a conversion of 83% and a turnover frequency of 20.6 × 103· h−1 in aqueous solution under mild conditions. The remarkable catalytic activity indicates that the nano-confinement of the Au(I) active site within the ferritin cage enhances its catalytic properties by stabilizing and solubilizing it. The less protected Au atom in the cysteine bridged dinuclear Au(I) active center was identified as critical for the Fr−Au cycloisomerases to catalyze this reaction. In addition, we provide insight into the catalytic mechanism through quantum chemical (QC) calculations, which reveal an energy barrier of 32.29 kJ/mol.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/10_pigeon_feather.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="10_pigeon_feather.png" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lu_site-resolved_2024" class="col-sm-8"> <div class="title">Site-resolved energetic information from HX/MS experiments</div> <div class="author"> <em>Chenlin Lu<sup>*</sup></em>, Malcolm L. Wells<sup>*</sup>, Andrew Reckers, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Anum Glasgow' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* These authors contributed equally to this work."> </i> </div> <div class="periodical"> Aug 2024 </div> <div class="periodical"> Pages: 2024.08.04.606547 Section: New Results </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2024.08.04.606547" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>While bioinformatics reveals patterns in protein sequences and structural biology methods elucidate atomic details of protein structures, it is difficult to attain equally high-resolution energetic information about protein conformational ensembles. We present PIGEON-FEATHER, a method for calculating free energies of opening (ΔGop) at single- or near-single-amino acid resolution for protein ensembles of all sizes from hydrogen exchange/mass spectrometry (HX/MS) data. PIGEON-FEATHER disambiguates and reconstructs all experimentally measured isotopic mass envelopes using a Bayesian Monte Carlo sampling approach. We applied PIGEON-FEATHER to reveal how E. coli and human dihydrofolate reductase orthologs (ecDHFR, hDHFR) have evolved distinct ensembles tuned to their catalytic cycles, and how two competitive inhibitors of ecDHFR arrest its ensemble in different ways. Extending the method to a large protein-DNA complex, we mapped ligand-induced ensemble reweighting in the E. coli lac repressor to understand the functional switching mechanism crucial for transcriptional regulation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/12_TFs_PBPs.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="12_TFs_PBPs.png" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="wells_conserved_2025" class="col-sm-8"> <div class="title">Conserved energetic changes drive function in an ancient protein fold</div> <div class="author"> Malcolm L. Wells<sup>*</sup>, <em>Chenlin Lu<sup>*</sup></em>, Daniel Sultanov<sup>*</sup>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Kyle C. Weber&lt;sup&gt;*&lt;/sup&gt;, Zhen Gong, Anum Glasgow' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> Apr 2025 </div> <div class="periodical"> Pages: 2025.04.02.646877 Section: New Results </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2025.04.02.646877" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Many protein sequences occupy similar three-dimensional structures known as protein folds. In nature, protein folds are well-conserved over the course of evolution, such that there are 100,000 times as many extant protein sequences than there are folds. Despite their common shapes, similar protein folds can adopt wide-ranging functions, raising the question: are protein folds so strongly conserved for the purpose of maintaining function-driving energetic changes in protein families? Here we show that a set of key energetic relationships in a family of bacterial transcription factors (TFs) is conserved using high-resolution hydrogen exchange/mass spectrometry, bioinformatics, X-ray crystallography, and molecular dynamics simulations. We compared the TFs to their anciently diverged structural homologs, the periplasmic binding proteins (PBPs), expecting that protein families that share the same fold and bind the same sugars would have similar energetic responses. Surprisingly, our findings reveal the opposite: the "energetic blueprints" of the PBPs and the TFs are largely distinct, with the allosteric network of the TFs evolving specifically to support the functional requirements of genome regulation, versus conserved interactions with membrane transport machinery in PBPs. These results demonstrate how the same fold can be adapted for different sense/response functions via family-specific energetic requirements - even when responding to the same chemical trigger. Understanding the evolutionarily conserved energetic blueprint for a protein family provides a roadmap for designing functional proteins de novo, and will help us treat aberrant protein behavior in conserved domains in disease-related drug targets, where engineering selectivity is challenging.\textless/p\textgreater</p> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%6C%75%63%6C%74%73%69%6E%67%68%75%61@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://scholar.google.com/citations?user=pnVz0cQAAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> </div> <div class="contact-note"></div> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Chenlin Lu. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>