<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Chenlin Lu </title> <meta name="author" content="Chenlin Lu"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://lucl13.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Chenlin</span> Lu </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/12_TFs_PBPs.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="12_TFs_PBPs.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="wells_conserved_2025" class="col-sm-8"> <div class="title">Conserved energetic changes drive function in an ancient protein fold</div> <div class="author"> Malcolm L. Wells<sup>*</sup>, <em>Chenlin Lu<sup>*</sup></em>, Daniel Sultanov<sup>*</sup>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Kyle C. Weber&lt;sup&gt;*&lt;/sup&gt;, Zhen Gong, Anum Glasgow' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* These authors contributed equally to this work."> </i> </div> <div class="periodical"> Apr 2025 </div> <div class="periodical"> Pages: 2025.04.02.646877 Section: New Results </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2025.04.02.646877" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Many protein sequences occupy similar three-dimensional structures known as protein folds. In nature, protein folds are well-conserved over the course of evolution, such that there are 100,000 times as many extant protein sequences than there are folds. Despite their common shapes, similar protein folds can adopt wide-ranging functions, raising the question: are protein folds so strongly conserved for the purpose of maintaining function-driving energetic changes in protein families? Here we show that a set of key energetic relationships in a family of bacterial transcription factors (TFs) is conserved using high-resolution hydrogen exchange/mass spectrometry, bioinformatics, X-ray crystallography, and molecular dynamics simulations. We compared the TFs to their anciently diverged structural homologs, the periplasmic binding proteins (PBPs), expecting that protein families that share the same fold and bind the same sugars would have similar energetic responses. Surprisingly, our findings reveal the opposite: the "energetic blueprints" of the PBPs and the TFs are largely distinct, with the allosteric network of the TFs evolving specifically to support the functional requirements of genome regulation, versus conserved interactions with membrane transport machinery in PBPs. These results demonstrate how the same fold can be adapted for different sense/response functions via family-specific energetic requirements - even when responding to the same chemical trigger. Understanding the evolutionarily conserved energetic blueprint for a protein family provides a roadmap for designing functional proteins de novo, and will help us treat aberrant protein behavior in conserved domains in disease-related drug targets, where engineering selectivity is challenging.\textless/p\textgreater</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/10_pigeon_feather.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="10_pigeon_feather.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lu_site-resolved_2024" class="col-sm-8"> <div class="title">Site-resolved energetic information from HX/MS experiments</div> <div class="author"> <em>Chenlin Lu<sup>*</sup></em>, Malcolm L. Wells<sup>*</sup>, Andrew Reckers, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Anum Glasgow' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* These authors contributed equally to this work."> </i> </div> <div class="periodical"> Aug 2024 </div> <div class="periodical"> Pages: 2024.08.04.606547 Section: New Results </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2024.08.04.606547" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>While bioinformatics reveals patterns in protein sequences and structural biology methods elucidate atomic details of protein structures, it is difficult to attain equally high-resolution energetic information about protein conformational ensembles. We present PIGEON-FEATHER, a method for calculating free energies of opening (Î”Gop) at single- or near-single-amino acid resolution for protein ensembles of all sizes from hydrogen exchange/mass spectrometry (HX/MS) data. PIGEON-FEATHER disambiguates and reconstructs all experimentally measured isotopic mass envelopes using a Bayesian Monte Carlo sampling approach. We applied PIGEON-FEATHER to reveal how E. coli and human dihydrofolate reductase orthologs (ecDHFR, hDHFR) have evolved distinct ensembles tuned to their catalytic cycles, and how two competitive inhibitors of ecDHFR arrest its ensemble in different ways. Extending the method to a large protein-DNA complex, we mapped ligand-induced ensemble reweighting in the E. coli lac repressor to understand the functional switching mechanism crucial for transcriptional regulation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/08_Rum55.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="08_Rum55.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="wang_enhancing_2024" class="col-sm-8"> <div class="title">Enhancing the stability of a novel D-allulose 3-epimerase from Ruminococcus sp. CAG55 by interface interaction engineering and terminally attached a self-assembling peptide</div> <div class="author"> Jing Wang, <em>Chenlin Lu</em>, Xuemei Shen, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Taibo He, Diannan Lu, Xiaoyan Wang, Yuan Zhang, Zhanglin Lin, Xiaofeng Yang' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>International Journal of Biological Macromolecules</em>, Jun 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.ijbiomac.2024.131986" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>D-allulose, a highly desirable sugar substitute, is primarily produced using the D-allulose 3-epimerase (DAE). However, the availability of usable DAE enzymes is limited. In this study, we discovered and engineered a novel DAE Rum55, derived from a human gut bacterium Ruminococcus sp. CAG55. The activity of Rum55 was strictly dependent on the presence of Co2+, and it exhibited an equilibrium conversion rate of 30.6 % and a half-life of 4.5 h at 50 â—¦C. To enhance its performance, we engineered the interface interaction of Rum55 to stabilize its tetramer structure, and the best variant E268R was then attached with a self-assembling peptide to form active enzyme aggregates as carrier-free immobilization. The half-life of the best variant E268R-EKL16 at 50 â—¦C was dramatically increased 30-fold to 135.3 h, and it maintained 90 % of its activity after 13 consecutive reaction cycles. Additionally, we identified that metal ions played a key role in stabilizing the tetramer structure of Rum55, and the dependence on metal ions for E268R-EKL16 was significantly reduced. This study provides a useful route for improving the thermostability of DAEs, opening up new possibilities for the industrial production of D-allulose.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/05_md_protein_cage.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="05_md_protein_cage.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lu_molecular_2023" class="col-sm-8"> <div class="title">Molecular Dynamics Simulation of Protein Cages</div> <div class="author"> <em>Chenlin Lu</em>, Xue Peng, and Diannan Lu </div> <div class="periodical"> <em>In Protein Cages: Design, Structure, and Applications</em>, Jun 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/978-1-0716-3222-2_16" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Molecular dynamics (MD) simulations enable the description of the physical movement of the system over time based on classical mechanics at various scales depending on the models. Protein cages are a particular group of different-size proteins with hollow, spherical structures and are widely found in nature, which have vast applications in numerous fields. The MD simulation of cage proteins is particularly important as a powerful tool to unveil their structures and dynamics for various properties, assembly behavior, and molecular transport mechanisms. Here, we describe how to conduct MD simulations for cage proteins, especially technical details, and analyze some of the properties of interest using GROMACS/NAMD packages.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/07_FrAu_ArM.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="07_FrAu_ArM.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lu_novel_2023" class="col-sm-8"> <div class="title">Novel Au(I)-Based Artificial Metallo-Cycloisomerase for Catalyzing the Cycloisomerization of Î³-Alkynoic Acids</div> <div class="author"> <em>Chenlin Lu</em>, Xue Peng, Basudev Maity, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Xiang Sheng, Yinhuan Zhou, Takafumi Ueno, Zheng Liu, Diannan Lu' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>ACS Catalysis</em>, Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/acscatal.3c01197" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Artificial metalloenzymes, which are designed rationally as hybrids of proteins and catalytically active transition-metal complexes, have become a promising approach for catalyzing unprecedented reactions for natural enzymes. In this study, we described the design and synthesis of an artificial metalloenzyme, a cycloisomerase that utilizes Au(I) incorporated into an apo-ferritin cage (Frâˆ’Au) to efficiently catalyze the cycloisomerization of alkynoic acids, with a conversion of 83% and a turnover frequency of 20.6 Ã— 103Â· hâˆ’1 in aqueous solution under mild conditions. The remarkable catalytic activity indicates that the nano-confinement of the Au(I) active site within the ferritin cage enhances its catalytic properties by stabilizing and solubilizing it. The less protected Au atom in the cysteine bridged dinuclear Au(I) active center was identified as critical for the Frâˆ’Au cycloisomerases to catalyze this reaction. In addition, we provide insight into the catalytic mechanism through quantum chemical (QC) calculations, which reveal an energy barrier of 32.29 kJ/mol.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/04_fr_4fold-Au.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="04_fr_4fold-Au.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lu_design_2022" class="col-sm-8"> <div class="title">Design of a gold clustering site in an engineered apo-ferritin cage</div> <div class="author"> <em>Chenlin Lu</em>, Basudev Maity, Xue Peng, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Nozomi Ito, Satoshi Abe, Xiang Sheng, Takafumi Ueno, Diannan Lu' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Communications Chemistry</em>, Dec 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s42004-022-00651-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Water-soluble and biocompatible protein-protected gold nanoclusters (Au NCs) hold great promise for numerous applications. However, design and precise regulation of their structure at an atomic level remain challenging. Herein, we have engineered and constructed a gold clustering site at the 4-fold symmetric axis channel of the apo-ferritin cage. Using a series of X-ray crystal structures, we evaluated the stepwise accumulation process of Au ions into the cage and the formation of a multinuclear Au cluster in our designed cavity. We also disclosed the role of key residues in the metal accumulation process. X-ray crystal structures in combination with quantum chemical (QC) calculation revealed a unique Au clustering site with up to 12 Au atoms positions in the cavity. Moreover, the structure of the gold nanocluster was precisely tuned by the dosage of the Au precursor. As the gold concentration increases, the number of Au atoms position at the clustering site increases from 8 to 12, and a structural rearrangement was observed at a higher Au concentration. Furthermore, the binding affinity order of the four Au binding sites on apo-ferritin was unveiled with a stepwise increase of Au precursor concentration.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/03_fr_au_diffusion.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="03_fr_au_diffusion.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="peng_synergistic_2021" class="col-sm-8"> <div class="title">The synergistic mechanisms of apo-ferritin structural transitions and Au(III) ion transportation: molecular dynamics simulations with the Markov state model</div> <div class="author"> Xue Peng<sup>*</sup>, <em>Chenlin Lu<sup>*</sup></em>, Zheng Liu, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Diannan Lu' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* These authors contributed equally to this work."> </i> </div> <div class="periodical"> <em>Physical Chemistry Chemical Physics</em>, Dec 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10/gnq2st" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Due to its unique structure, recent years have witnessed the use of apo-ferritin to accumulate various non-natural metal ions as a scaffold for nanomaterial synthesis. However, the transport mechanism of metal ions into the cavity of apo-ferritin is still unclear, limiting the rational design and controllable preparation of nanomaterials. Here, we conducted all-atom classical molecular dynamics (MD) simulations combined with Markov state models (MSMs) to explore the transportation behavior of Au(III) ions. We exhibited the complete transportation paths of Au(III) from solution into the apo-ferritin cage at the atomic level. We also revealed that the transportation of Au(III) ions is accompanied by coupled protein structural changes. It is shown that the 3-fold axis channel serves as the only entrance with the longest residence time of Au(III) ions. Besides, there are eight binding clusters and five 3-fold structural metastable states, which are important during Au(III) transportation. The conformational changes of His118, Asp127, and Glu130, acting as doors, were observed to highly correlate with the Au(III) ionâ€™s position. The MSM analysis and Potential Mean Force (PMF) calculation suggest a remarkable energy barrier near Glu130, making it the rate-limiting step of the whole process. The dominant transportation pathway is from cluster 3 in the 3-fold channel to the inner cavity to cluster 5 on the inner surface, and then to cluster 6. These findings provide inspiration and theoretical guidance for the further rational design and preparation of new nanomaterials using apo-ferritin.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/02_MHETase.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="02_MHETase.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="peng_distal_2021" class="col-sm-8"> <div class="title">A distal regulatory strategy of enzymes: from local to global conformational dynamics</div> <div class="author"> Xue Peng<sup>*</sup>, <em>Chenlin Lu<sup>*</sup></em>, Jian Pang, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Zheng Liu, Diannan Lu' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* These authors contributed equally to this work."> </i> </div> <div class="periodical"> <em>Physical Chemistry Chemical Physics</em>, Dec 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10/gnp5nn" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Modulating the distribution of various states in protein ensembles through distal sites may be promising in the evolution of enzymes in desired directions. , Modulating the distribution of various states in protein ensembles through distal sites may be promising in the evolution of enzymes in desired directions. However, the prediction of distal mutation hotspots that stabilize the favoured states from a computational perspective remains challenging. Here, we presented a strategy based on molecular dynamics (MD) and Markov state models (MSM) to predict distal mutation sites. Extensive MD combined with MSM was applied to determine the principally distributed metastable states interconverting at a slow timescale. Then, molecular docking was used to classify these states into active states and inactive ones. Distal mutation hotspots were targeted based on comparing the conformational features between active and inactive states, where mutations destabilize the inactive states and show little influence on the active state. The proposed strategy was used to explore the highly dynamic MHETase, which shows a potential application in the biodegradation of poly(ethylene terephthalate) (PET). Seven principally populated interrelated metastable states were identified, and the atomistic picture of their conformational changes was unveiled. Several residues at distal positions were found to adopt more H-bond occupancies in inactive states than active states, making them potential mutation hotspots for stabilizing the favoured conformations. In addition, the detailed mechanism revealed the significance of calcium ions at a distance from the catalytic centre in reshaping the free energy landscape. This study deepens the understanding of the conformational dynamics of Î±/Î² hydrolases containing a lid domain and advances the study of enzymatic plastic degradation.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/01_lipase_msm.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="01_lipase_msm.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lu_global_2020" class="col-sm-8"> <div class="title">Global and Kinetic Profiles of Substrate Diffusion in \textitCandida antarctica Lipase B: Molecular Dynamics with the Markov-State Model</div> <div class="author"> <em>Chenlin Lu</em>, Xue Peng, Diannan Lu, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Zheng Liu' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>ACS Omega</em>, May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10/gnqrm3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Profiling substrate diffusion pathways with kinetic information, which accounts for the dynamic nature of enzymeâˆ’substrate interaction, can enable molecular reengineering of enzymes and process optimization of enzymatic catalysis. Candida antarctica lipase B (CALB) is extensively used for producing various chemicals because of its rich catalytic mechanisms, broad substrate spectrum, thermal stability, and tolerance to organic solvents. In this study, an all-atom molecular dynamics (MD) combined with Markov-state models (MSMs) implemented in pyEMMA was proposed to simulate diffusion pathways of 4-nitrophenyl ester (4NPE), a commonly used substrate, from the surface into the active site of CALB. Six important metastable conformations of CALB were identified in the diffusion process, including a closed state. An induced-fit mechanism incorporating multiple pathways with molecular information was proposed, which might find unprecedented applications for the rational design of lipase for green catalysis.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> Â© Copyright 2025 Chenlin Lu. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>